src/main.cpp

```diff
@@ -45,6 +45,7 @@
 #include "config_compp.h"
 #include "config_effects.h"
 #include "lvl_script.h"
+#include "lvl_script_conditions.h"
 #include "thing_list.h"
 #include "player_instances.h"
 #include "player_utils.h"
```

Comment:
github doesn't allow me to put comment on right line but I see 3 _DK_tag_cursor_blocks_ dllimports left over unused, could you throw those out